CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [0.2.1]
+### Fixed
+* Inaccurate generation of burst bounding boxes in a Qinghai China example pair
```

Comment:
I'm not sure the Changelog is the best place to call out the particular example that failed. Is this the pair used in the command that you commented [here](https://github.com/ASFHyP3/hyp3-isce2/pull/36#issuecomment-1518208625)? I think that comment is probably enough (or you could document it in an issue if you want to make it more visible).

Reply:
Also, is this part of the same change as the bullet under `Added`? We could just combine these into a single bullet.

Reply:
Oh sorry, now I'm realizing this probably refers to the changes in [`download_example.py`](https://github.com/ASFHyP3/hyp3-isce2/pull/36/files#diff-1c2b82e1309d796ea81d82e401a27118fc3e3dfd564fe269e989f783dce2fa35)?

Reply:
Nope, your second comment was correct. I've updated the changelog.